Libraries/Animated/nodes/AnimatedStyle.js

```diff
@@ -50,8 +62,17 @@ export default class AnimatedStyle extends AnimatedWithChildren {
     return updatedStyle;
   }
 
-  __getValue(): Object {
-    return this._walkStyleAndGetValues(this._style);
+  __getValue(): Array<Object> {
+    if (Platform.OS === 'web') {
+      return [this._inputStyle, this._walkStyleAndGetValues(this._style)];
+    }
+
+    return [
```

Comment:
Can we keep this as `return this._walkStyleAndGetValues(this._style)` for native?

Reply:
Yes, let's avoid an additional array here if it's not necessary.

Reply:
Sounds good, I've just pushed a commit updating this, please let me know if I should change something else as well